src/frontend/js/modules/datetime/datetime.run.js

```diff
@@ -4,11 +4,17 @@ require('./datetime.service.js');
   'use strict';
 
   angular.module('esn.datetime')
+    .config(function(tmhDynamicLocaleProvider) {
+      tmhDynamicLocaleProvider.localeLocationPattern('i18n/angular-locale_{{locale}}.js');
```

Comment:
```suggestion
      tmhDynamicLocaleProvider.localeLocationPattern('node_modules/angular-i18n/angular-locale_{{locale}}.js');
```

Does this work?

Reply:
no that does not work ,i had try this on the first , webpack is not able to find files , forthat i find a solution with copy-webpack-plugin to copy files under our main directory